.formatting/Manifest.toml

```diff
@@ -0,0 +1,226 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Is it wanted, that his file is in the PR?

Reply:
Removed in 8321d60 .